pkg/console/install_panels.go

```diff
@@ -1388,6 +1388,11 @@ func addNetworkPanel(c *Console) error {
 			mgmtNetwork.Gateway = ""
 			mgmtNetwork.MTU = 0
 		}
+
+		if !isDefaultRouteExist() {
+			return fmt.Sprint("No default route found. Please check the router setting on the DHCP server"), nil
```

Comment:
This can just be `return "No default route found. Please check the router setting on the DHCP server", nil` (there's no need for the `fmt.Sprint`, because there's no other variables being interpolated into the string).